### ISSUE TITLE
Some questions do not fit on the screen

### ISSUE THREAD

**Issue by BieleckiLtd** [KristofferStrube/EditorConfigWizard#13]
![IMG_6331](https://github.com/KristofferStrube/EditorConfigWizard/assets/57046415/4cdb4629-f037-42f8-8d53-b0be22a88b0a)

**Comment by KristofferStrube** [comment]
Makes sense. We will try to fix this. This is specifically for the Sample-Fixed formats.
